Mathlib/RingTheory/Artinian.lean

```diff
@@ -471,4 +452,10 @@ instance : IsArtinianRing (Localization S) :=
 
 end Localization
 
+instance isMaximal_of_isPrime (p : Ideal R) [p.IsPrime] : p.IsMaximal :=
+  Ideal.Quotient.maximal_of_isField _ <|
+    (MulEquiv.ofBijective _ ⟨IsFractionRing.injective (R ⧸ p) (FractionRing (R ⧸ p)),
+      localization_surjective (nonZeroDivisors (R ⧸ p)) (FractionRing (R ⧸ p))⟩).isField _
```

Comment:
Is this `MulEquiv` a useful standalone definition?

Reply:
It's actually a RingEquiv between R/p and its fraction field, but it's just saying R/p is a field (and we get a field instance from the def [Ideal.Quotient.field](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Ideal/Quotient.html#Ideal.Quotient.field)), so I think it isn't very useful.